librz/core/cbin.c

```diff
@@ -37,33 +37,54 @@ static void pair(const char *key, const char *val) {
 	if (!val || !*val) {
 		return;
 	}
-	rz_cons_printf("%-" PAIR_WIDTH "s%s\n", key, val);
 }
 
-static void pair_bool(PJ *pj, const char *key, bool val) {
+static char *get_coloured(const char *flag) {
+	if (strncmp(flag, "true", 5)) {
+		return rz_str_newf(Color_GREEN "true");
+	} else {
+		return rz_str_newf(Color_RED "false");
+	}
+}
+
+static void pair_bool(RzTable *t, PJ *pj, const char *key, bool val, RzList *row_list) {
+	RzTableColumnType *typeString = rz_table_type("bool");
```

Comment:
if this is only used on the table, please move it in the else scope.

Reply:
@sandrabeme please don't forget this one.